docs/sections/settings.rst

```diff
@@ -267,4 +267,13 @@ It is recommended to set this to ``'filesystem'`` for new applications if possib
 
 .. note::
     When using ``'filesystem'`` with the default file storage, or another local file storage, you will
-    need to sync RSA keys between your application servers.
\ No newline at end of file
+    need to sync RSA keys between your application servers.
+
+OIDC_REFRESH_TOKEN_ALIVE_HOOK
+=============================
+
+.. currentmodule:: oidc_provider.settings
+.. autoattribute:: DefaultSettings.OIDC_REFRESH_TOKEN_ALIVE_HOOK
+
+.. currentmodule:: oidc_provider.lib.utils.common
+.. autofunction:: default_refresh_token_alive_hook
```

Comment:
I'm not sure I'm happy with how this documentation renders. It does have the advantage of not having to duplicate the docs between `settings.py` and `settings.rst`.